### PR TITLE
Monochrome workflow

### PR DESCRIPTION
**Pull request by matt-maguire**
update various places to accomodate new monochrome workflow.

In the new screenshot, I excluded the title bar of the dialogue box, as this is dependent on the window manager and OS.

### CODE REVIEW

**Comment by matt-maguire** [comment]
looks like this feature is still under development, just saw new PR#6516

**Comment by elstoc** [comment]
Should we maybe keep this PR as WIP as well then?

**Comment by matt-maguire** [comment]
probably wouldn't hurt

**Comment by elstoc** [comment]
Ok we'll wait for action on the main project before reviewing then

**Comment by elstoc** [comment]
Sorry I forgot all about this one and added a replacement presets screenshot of my own. Happy if you want to overwrite it with this one.

**Comment by matt-maguire** [comment]
I rebased the PR and dropped my screenshot commit (yours has the titlebar, whereas mine didn't).

**Comment by elstoc** [comment]
Ok thanks. Does this include the changes for #6516 now or are those still TBC?

**Comment by matt-maguire** [comment]
Not yet, let me update then we can merge.

**Comment by matt-maguire** [comment]
Looks like there is a new PR darktable-org/darktable#6542 which implements some sort of undo function for monochrome (the previous attempt at that was aborted in darktable-org/darktable#6512 when problems were found).

**Comment by matt-maguire** [comment]
Undo feature for monochrome flags is still under development. When it becomes available, we can add "monochrome flags" to the list of things that can be undone in the undo-redo section. Otherwise, I think this should be ready to merge.

File: content/overview/workflow/import-rate-tag.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ import images from the filesystem
...
 
-Once images are imported, they will appear in the lighttable view. By default, the images will all be given a one-star rating. 
+Once images are imported, they will appear in the lighttable view. By default, the images will all be given a one-star rating.
+
+The photo will auto-tagged with `darktable|mode|monochrome` if it was taken on a monochrome camera, if the demosaic module is in passthrough, or if [`preferences` -> `processing` -> `detect monochrome previews`](../../../preferences-settings/processing.md) is enabled and the embedded preview jpeg is monochrome. If you have permanent overlays enabled on your lighttable thumbnails, it will also place a `B&W` indicator next to the file type. The automatic tag informs modules that the image is intended to be monochrome, and they can perform special monochrome-specific handling where needed.
```
Comment (jenshannoschwalm) [inline content/overview/workflow/import-rate-tag.md:19]: The photo will be auto-tagged ...

Comment (matt-maguire) [reply content/overview/workflow/import-rate-tag.md]: thanks, pushed a correction

**Comment by elstoc** [comment]
@matt-maguire Is this ready for review or is it still WIP?

**Comment by matt-maguire** [comment]
It's ready.

**Comment by elstoc** [comment]
Ok. Two comments

- If we're going to talk about images "being identified as monochrome" in the user manual, we really need a section where this is formally defined and links to that section wherever you mention it. I see you've put something in the workflow overview section but I'm not sure this is the best place. This section should be a more high level overview of things that are discussed more formally elsewhere. I'm happy to take suggestions on where this should be put.
- Regarding the undo/redo section, I'm not sure that this is supposed to be an exhaustive list. I'd be tempted to reduce the list rather than increase it, since, the more things are mentioned the more it looks like it _is_ supposed to be exhaustive.

**Comment by matt-maguire** [comment]
So I moved the monochrome info out of the import workflow overview section, and created a new section in the "Guides and Tutorials" section. This now covers the importing of monochrome, and some information of the physical vs perceptual approach to converting from colour to B&W.

**Comment by elstoc** [comment]
Looks good to me. Thanks

**Comment by matt-maguire** [comment]
Ok, but I've messed up the description of the demosaic passthorugh (monochrome) feature, we'll need to correct it.

**Pull request merged**